src/bundle.c

```diff
@@ -838,7 +838,6 @@ static int install_bundles(struct list *bundles, struct list **subs, struct mani
 
 	mom->files = files_from_bundles(mom->submanifests);
 	mom->files = consolidate_files(mom->files);
-	timelist_timer_stop(global_times);
```

Comment:
I'm reviewing this again and I don't understand why did you remove that stop. In that section we are counting how long does it take to download missing files and we are end on downloading after this step.

Reply:
I had added comments on what timeliest_timer_stop was closing each timeliest_timer_start on my other commit that made it easier to see it. But that was in the commit I had for prettifying the output as tree, so I removed that. I will re-add those comments so we can more easily identify those changes.

Reply:
so one of the timelist_timer_stop was extra in this section of the code, either the one at line 837 or the one at line 841. Maybe I should have removed the one at 837 instead of the one at 841 so it includes the file consolidation. @otaviobp should I move the timelist_timer_stop to line 841 instead?